languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -51559,336 +51559,170 @@ USA
-                </token>
-            </antipattern>
-            <antipattern>
-                <token inflected="yes">be</token>
-                <token chunk_re="B-NP.*" />
-                <token min="0" chunk_re="E-NP.*" />
-                <token chunk="B-PP" />
-            </antipattern>
-            <antipattern>
-                <token>for</token>
-                <token>which</token>
-                <token chunk_re="B-NP.*" />
-                <token inflected="yes">be</token>
-            </antipattern>
-            <antipattern>
-                <token postag="VBP" />
-                <token postag="VBG|VBD|VBN" postag_regexp="yes" />
-            </antipattern>
-            <antipattern>
-                <token postag="VB|VBP" postag_regexp="yes" />
-                <token regexp="yes">able|ful|&apostrophe;</token>
-            </antipattern>
-            <antipattern>
-                <token regexp="yes">need|you|the</token>
-                <token regexp="yes">no|'re|Brackets</token>
-                <token regexp="yes">longer|dad|feature</token>
-            </antipattern>
-            <antipattern>
-                <token regexp="yes">chances|odds|problems?</token>
-                <token inflected="yes">be</token>
-            </antipattern>
-            <antipattern>
-                <token postag="VBZ" />
-                <token postag="VBZ" />
-            </antipattern>
-            <antipattern>
-                <token postag="VBD" />
-                <token postag="VBD" />
-            </antipattern>
-            <antipattern>
-                <token regexp="yes">garbage|trash</token>
-                <token>can</token>
-            </antipattern>
+        <rulegroup id="BE_PART_AGREEMENT_2" name="Were they send (sent) via WhatsApp?" default="temp_off">
             <rule>
                 <antipattern>
-                    <token>it</token>
-                    <token postag="N.*" postag_regexp="yes" />
-                    <token postag="CC" />
-                </antipattern>
-                <antipattern>
-                    <token postag="PRP" />
-                    <token inflected="yes">be</token>
-                    <token postag="PRP" />
+                    <token chunk_re="E-NP.*" />
+                    <token regexp="yes" case_sensitive="yes">[A-Z].*</token>
                 </antipattern>
                 <antipattern>
-                    <token regexp="yes">am|are</token>
-                    <token regexp="yes">I|you|we</token>
-                    <token regexp="yes" inflected="yes">be|think</token>
+                    <token chunk_re="E-NP.*" />
+                    <token chunk="B-ADVP" />
                 </antipattern>
                 <antipattern>
-                    <token>is</token>
-                    <token regexp="yes">I|you|we|they</token>
+                    <token chunk_re="E-NP.*" />
+                    <token regexp="yes">.*'d</token>
```

Comment:
`'d` is always going to be tokenized as its own token – so this AP cannot match…